pubsub/topics/main.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command topics is a tool to manage Google Cloud Pub/Sub topics by using the Pub/Sub API.
+// See more about Google Cloud Pub/Sub at https://cloud.google.com/pubsub/docs/overview.
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	// [START imports]
+	"golang.org/x/net/context"
+
+	"cloud.google.com/go/pubsub"
+	// [END imports]
+)
+
+func main() {
+	ctx := context.Background()
+	// [START auth]
+	proj := os.Getenv("GCLOUD_PROJECT")
+	if proj == "" {
+		fmt.Fprintf(os.Stderr, "GCLOUD_PROJECT environment variable must be set.\n")
+		os.Exit(1)
+	}
```

Comment:
Take project as a parameter?
We should probably standardize here. What do you lean towards?